lib/ansible/module_utils/ansible_tower.py

```diff
@@ -45,31 +47,39 @@ def tower_auth_config(module):
     it will attempt to fetch values from the module pararms and
     only pass those values that have been set.
     '''
+    auth_config = {}
+
+    settings = Settings()
+    # XXX use all parsers ? possibly allow user to choose ?
+    settings._parser_names = ['ansible', 'environment', 'defaults']
```

Comment:
by not include the 'local' or 'user' option in the list, users won't be able to use these modules by having it automatically pick up their existing config file in their user directory or elsewhere.

I am not really sure if that was a bug or a feature of the prior implementation...